app/src/main/res/layout/main_activity.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
ConstraintLayout is better for flexible UI

Reply:
What tech debt reduction do you achieve in moving from Jetpack Compose to an XML based layout?